src/main/java/core/basesyntax/strategy/handlers/BalanceOperationHandler.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.strategy.handlers;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperationHandler implements OperationHandler {
+    private static final String RUNTIME_EXCEPTION_MESSAGE = "Quantity can't be negative";
```

Comment:
too generic naming for this field, it should just describe the exception message